superchain/configs/sepolia/tbn.toml

```diff
@@ -8,8 +8,9 @@ batch_inbox_addr = "0xFf00000000000000000000000000000000042069"
 canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
 ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
-fjord_time = 1723204838  #Saturday June 8 2024 10:47:18 AM UTC
```

Comment:
Strange that these timestamp comments incorrect before -- @bitwiseguy do we know why?

Reply:
The old ones look like they were created manually, not with `date -u -r`.